tests/test_api_groups_get.py

```diff
@@ -106,6 +106,31 @@ def test_sort_by_name(db_create_group, api_get, order_how_query, reverse_list):
         assert response_data["results"][idx]["id"] == group_id_list[idx]
 
 
+@pytest.mark.parametrize("order_how", ["ASC", "DESC"])
+def test_sort_by_updated_time(db_create_group, api_get, order_how):
+    num_groups = 5
+    sort_query = f"?order_by=updated&order_how={order_how}"
+
+    _ = [db_create_group(f"testGroup{idx}") for idx in range(num_groups)]
```

Comment:
This statement is assigning the resulting value to an unnamed variable that doesn't get used later, so a for-loop makes more sense than a list comprehension here:
```suggestion
    for idx in range(num_groups): db_create_group(f"testGroup{idx}")
```